docs/ttl.md

```diff
@@ -15,7 +15,22 @@ metadata:
   ...
 ```
 
-TTL must be a positive integer encoded as string.
+In the example above TTL is specified as an integer encoded as string representing seconds. 
```

Comment:
Do you mind adding this in line 4, this should also count for ingress?

Reply:
Sorry, I don't quite get you, could you please elaborate on this?

Reply:
Instead of writing it on line 18, I would put it into line 4 where Ingress annotation is explained.

Reply:
Done